src/test/java/edu/indiana/dlib/amppd/controller/BundleItemControllerTests.java

```diff
@@ -81,4 +85,54 @@ public void shouldSaveUploadedBundleSupplement() throws Exception {
 						jsonPath("$.items[0].id").doesNotExist());    	
     }
 
+    @Test
+    public void shouldNotAddDuplicateItemToBundle() throws Exception {   	
+    	Item item = new Item();
+    	item.setId(1l);
+    	item.setBundles(new HashSet<Bundle>());
+    	
+    	Bundle bundle = new Bundle();
+    	bundle.setId(1l);
+    	bundle.setItems(new HashSet<Item>());
+    	
+    	item.getBundles().add(bundle);
+    	bundle.getItems().add(item);    	
```

Comment:
Same thing here. I believe adding bundle to item should be sufficient. But I might be wrong. We can discuss this.